test/quantization/test_quant_primitives.py

```diff
@@ -863,6 +864,39 @@ def test_choose_qparams_and_quantize_scale_only_sinq(self):
         ).reshape(input.shape)
         self.assertFalse(torch.isnan(reconstructed).any())
 
+    def test_choose_qparams_and_quantize_scale_only_hqq(self):
+        """Test HQQ quantization produces valid outputs with correct shapes and ranges."""
+        torch.manual_seed(self.SEED)
+        input = torch.randn(128, 256, dtype=torch.float32)
+        block_size = [1, 64]
+        qmin = -(2 ** (4 - 1))
+        qmax = 2 ** (4 - 1) - 1
+
+        qdata, scale = _choose_qparams_and_quantize_scale_only_hqq(
+            input,
+            block_size=block_size,
+            qmin=qmin,
+            qmax=qmax,
+            iters=20,
+        )
+
+        # Check quantized data shape and dtype
```

Comment:
test doesn't look very hard to pass? would it be better to make it stronger by comparing with the results of https://github.com/pytorch/ao/blob/8b2367bda34af74d739bc7ffda06942a7c069763/torchao/quantization/quant_primitives.py#L34?